src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Concat.txt

```diff
@@ -111,3 +111,24 @@ Errors: Error 43-48: Name isn't valid. 'Value' isn't recognized.
 // Errors in the lambda
 >> Concat([2, 1, 0, -1, -2], Text(1 / Value), ",")
 Error({Kind:ErrorKind.Div0})
+
+// Coercions on lambda
```

Comment:
These tests are cases that failed before and now they pass?

It seems reasonably, but it's technically a change in semantics, we should get a confirmation from Greg.